### ISSUE TITLE
Importing macro breaks import of other types

### ISSUE THREAD

**Issue by AndreKR** [intellij-rust/intellij-rust#5668]
## Environment

* **IntelliJ Rust plugin version:** 0.2.123.3132-193
* **Rust toolchain version:** 1.45.0 (nightly)
* **IDE name and version:** CLion 2019.3.5
* **Operating system:** Win 7

## Problem description
This code is fine:

```rust
use anyhow::Result;

fn foo() -> Result<()> {
    Ok(())
}
```
![image](https://user-images.githubusercontent.com/1188538/85955652-57e80f80-b980-11ea-9cda-7be4d133000b.png)

But this code shows "Unresolved Reference":
```rust
use anyhow::Result;
use anyhow::anyhow;

fn foo() -> Result<()> {
    Err(anyhow!("This is an error"))
}
```
![image](https://user-images.githubusercontent.com/1188538/85955707-d2b12a80-b980-11ea-8e2e-a752536d7b3a.png)

**Comment by Undin** [comment]
It's duplicate of #4544 and the issue is already fixed in #5603
You can try the latest [nightly](https://intellij-rust.github.io/install#nightly-builds) build or wait for next release

**Comment by Undin** [comment]
@AndreKR Note, you are using CLion 2019.3 that we no longer support. So release builds with the corresponding fix will be available only for 2020.1 and 2020.2 IDEs

**Comment by AndreKR** [comment]
Updated to 2020.1.2 and switched the Rust plugin to nightly. Living on the edge now. :) The error is gone. :+1:
